regress/geography_expected

```diff
@@ -27,3 +27,4 @@ geog_precision_pazafir|0|0
 #2422|1|1609|t|t|1400.230|1396.816|1400.230|1400.230
 #2422|1|1600|t|t|1400.230|1396.816|1400.230|1400.230
 #2422|1|1068|f|f|1400.230|1396.816|1400.230|1400.230
+segmentize_geography|t
```

Comment:
I think it would be better to give a rounded number than a boolean value.
Otherwise a max distance of 0 would still not be caught as a failure